p5js/coding-challenges/solar-system-3d/solar_system_3d.js

```diff
@@ -4,6 +4,7 @@ function setup(){
   createCanvas(windowWidth, windowHeight);
 
   system = new SolarSystem();
+  system.init();
```

Comment:
Allowing the `system` object to be instantiated before calling `init()` allows the `MassiveObject` to utilize the global system variable in its initalization.

Could have taken different approaches:
- Shift to global `Mode` object and pull the `2D` mode out from `SolarSystem`
- Pass the `SolarSystem` object into the `MassiveObject` constructor ...